app/src/main/java/org/sil/storyproducer/controller/storylist/StoryPageFragment.kt

```diff
@@ -0,0 +1,151 @@
+package org.sil.storyproducer.controller.storylist
+
+import android.app.Activity
+import android.content.Context
+import android.os.Build
+import android.os.Bundle
+import android.text.Html
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentTransaction
+import org.sil.storyproducer.R
+import org.sil.storyproducer.activities.BaseActivity
+import org.sil.storyproducer.controller.MainActivity
+import org.sil.storyproducer.model.Story
+import org.sil.storyproducer.service.SlideService
+
+
+class StoryPageFragment : Fragment() {
```

Comment:
Could you add a comment about the scope of this class - is this what it does??
"StoryPageFragment implements one tab on the story list page"

Reply:
Done! Hopefully it makes a little more sense now.